package/lib/src/utils.dart

```diff
@@ -180,4 +181,17 @@ abstract class Utils {
       return blueprint.hasMatch(exact.toString());
     }
   }
+
+  static RegExp tryCastToRegExp(dynamic pathBlueprint) {
```

Comment:
We could add a doc comment here, nothing much, just so we don't degrade documentation coverage.